src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -5,79 +5,193 @@ class Buff:
     Type = None
     Title: str = None
 
+class PyBaseControl:
+
+    def Add(self, childControl: PyControl) -> None:
```

Ruff (0.13.1)

Undefined name `PyControl`

(F821)